### PR TITLE
Add conda package recipe

### PR DESCRIPTION
**Pull request by jonmmease**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ install:
   - conda config --set always_yes yes --set changeps1 no
   - conda update -q conda
   - conda info -a
-  - conda create -q -n test-environment -c conda-forge python=$TRAVIS_PYTHON_VERSION pandas dask numba numpy "pyarrow>=0.15" pytest hypothesis scipy shapely geopandas
+  - conda create -q -n test-environment -c conda-forge python=$TRAVIS_PYTHON_VERSION pandas dask numba numpy "pyarrow>=0.15" pytest hypothesis scipy shapely geopandas param
   - conda activate test-environment
   # hilbertcurve not available with conda, install with pip
```
Comment (xhochy) [inline .travis.yml]: I contributed hilbertcurve yesterday, you can now also conda install it.

File: conda.recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    {% for group,epoints in sdata.get("entry_points",{}).items() %}
+      {% for entry_point in epoints %}
+      - {{ entry_point }}
+      {% endfor %}
+      {% endfor %}
+
+requirements:
+  build:
```
Comment (xhochy) [inline conda.recipe/meta.yaml]: This should be `host` in modern-day conda.

File: .travis.yml (hunk 1 above)
Comment (jonmmease) [reply .travis.yml]: Oh awesome. Thanks!

File: conda.recipe/meta.yaml (hunk 2 above)
Comment (jonmmease) [reply conda.recipe/meta.yaml]: :+1:

**Pull request merged**